### PR TITLE
Added autoexec sas for modern pipeline. Updated SAS Program reference…

### PR DESCRIPTION
**Pull request by rseshadri99**
Added autoexec sas for modern pipeline. Updated SAS Program reference… to point to correct data sources for new pipeline

### CODE REVIEW

File: source/sas/dw/etl/src_modern/MasterEtl1 - Copy.sas
Comment (upasanapattnaik-eq) [inline source/sas/dw/etl/src_modern/MasterEtl1 - Copy.sas:1]: Is this file needed?

File: source/sas/dw/etl/rdbdata_modern/activity_log_detail.sas7bdat
Comment (upasanapattnaik-eq) [inline source/sas/dw/etl/rdbdata_modern/activity_log_detail.sas7bdat:1]: Are the .sas7bat files required? Would any of these contain any PII data?

File: source/sas/dw/etl/src_modern/MasterEtl1 - Copy.sas
Comment (rseshadri99) [reply source/sas/dw/etl/src_modern/MasterEtl1 - Copy.sas:1]: some previous updates - if we need to refer

File: source/sas/dw/etl/rdbdata_modern/activity_log_detail.sas7bdat
Comment (rseshadri99) [reply source/sas/dw/etl/rdbdata_modern/activity_log_detail.sas7bdat:1]: the data is anonymized

**Comment by upasanapattnaik-eq** [comment]
LGTM

**Review by upasanapattnaik-eq — APPROVED** [review]
